front-end/src/pages/Search.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useContext } from "react";
+import Container from "react-bootstrap/Container";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Nav from "react-bootstrap/Nav";
+import InputGroup from "react-bootstrap/InputGroup";
+import FormControl from "react-bootstrap/FormControl";
+import {FormattedMessage} from 'react-intl';
+import "../style.css";
+import logo from "../img/logo.svg";
+import Theme from "../components/Theme";
+
+import {Context} from "../components/lang/LanguageWrapper";
+
+
+const Search = () => {
+
+  const langContext = useContext(Context);
+  const currentLang = langContext.locale;
+
+  const [searchQuery, setSearchQuery] = useState("");
+  const [searchResults, setsearchResults] = useState("");
+
+  const updateQuery = (e) => {
+    setSearchQuery(e.target.value);
+  }
+
+  const submitQuery = async() => {
+    /* Single search term: https://dev.handshape.com/search?df=text_en_txt&q=fish */
+    const protocol = "https://";
+    //const currentHost = window.location.hostname;
+    const currentHost = "dev.handshape.com";
```

Comment:
This looks an awful lot like a dev-time configuration key that got committed. Same with the comment on line 30.